### PR TITLE
Issue #3208355: Use asynchronous widget for metatag handling

### PR DESCRIPTION
**Pull request by chrfritsch**
Make sure these boxes are checked before submitting your pull request - thank you!

- [ ] All coding styles are fulfilled. ([How to check for cs issues?](https://github.com/BurdaMagazinOrg/thunder-dev-tools/blob/master/README.md#code-style-guidelines))
- [ ] All tests are running locally. ([How to run the test?](https://github.com/thunder/thunder-distribution/blob/8.x-4.x/docs/development.md#how-to-run-the-tests))
- [ ] Necessary update hooks are provided.
- [ ] User roles have correct access for new introduced permission.
- [ ] Every thunder module has a README.md in its root. Follow [this guidelines](https://www.drupal.org/node/2181737), but we don't need every topic.
- [ ] Code is covered with well-balanced amount of inline comments.

If you are really awesome, then your feature is covered by additional tests. Well done!

### CODE REVIEW

**Comment by alexpott** [comment]
Do we want to leave existing sites alone? Or do we want to use the thunder_update functionality?

**Comment by chrfritsch** [comment]
Sure, I was not done. The baby needed me to go for a walk 😉

File: tests/src/FunctionalJavascript/ThunderArticleTestTrait.php [hunk 1]
```diff
@@ -28,6 +28,11 @@ public function articleFillNew(array $fieldValues) {
 
     if (!empty($fieldValues)) {
       $this->expandAllTabs();
+      if ($this->getSession()->getPage()->hasButton('Customize meta tags')) {
+        $this->getSession()->getPage()->pressButton('Customize meta tags');
+        $this->assertSession()->assertWaitOnAjaxRequest();
+        $this->expandAllTabs();
```
Comment (alexpott) [inline tests/src/FunctionalJavascript/ThunderArticleTestTrait.php:34]: expandAllTabs is a funny function.

**Review by dbosen — APPROVED** [review]
lgtm

**Pull request merged**